mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -246,7 +246,12 @@ static void populateDefaults() {
       paddingHeight.setValue(0);
       paddingWidth.setValue(0);
       num_cu.setValue(120);
+      arch.setValue("gfx908");
     }
+  }
+
+  if (xdlopsV2.getValue() == true) {
+    num_cu.setValue(120);
     arch.setValue("gfx908");
   }
```

Comment:
You shouldn't need this additional if condition statement. The else condition above already set `num_cu` and `arch`.

Reply:
This covers the case when -p is not present